timemachine/fe/atom_mapping.py

```diff
@@ -323,6 +323,41 @@ def core_bonds_broken_count(mol_a, mol_b, core):
     return count
 
 
+def get_invalid_chiral_conversion(
+    bond_graph: nx.Graph, disabled_bonds: Set[Tuple[int, int]]
+) -> Optional[Tuple[int, List[int]]]:
+    for node in bond_graph.nodes():
+        nbs = list(nx.neighbors(bond_graph, node))
+        # TODO: handle X3 chiral centers
+        if len(nbs) == 4:
+            disabled_bonds_count = sum(1 for nb in nbs if canonicalize_bond((node, nb)) in disabled_bonds)
+            if disabled_bonds_count > 1:
+                return node, nbs
+    return None
+
+
+def _graph_fails_chiral_assertion(
+    bond_graph: nx.Graph, core_nodes: Set[int], core_disabled_bonds: Set[Tuple[int, int]]
+) -> bool:
+    """A graph fails the chiral assertion if every choice of dummy group assignments fails.
+
+    A particular dummy group assignment fails if there exists a 4-connected atom with more than a single broken bond.
+    """
+    if len(core_nodes) == bond_graph.number_of_nodes() and get_invalid_chiral_conversion(
+        bond_graph, core_disabled_bonds
+    ):
+        return True
+
+    # return False if there is _any_ chirally-valid dummy group assignment
+    for dga in generate_dummy_group_assignments(bond_graph, core_nodes):
+        dga_disabled_bonds = compute_disabled_bonds_in_dga(bond_graph, core_nodes, dga)
+        disabled_bonds = dga_disabled_bonds.union(core_disabled_bonds)
+        if not get_invalid_chiral_conversion(bond_graph, disabled_bonds):
```

Comment:
```suggestion
        if get_invalid_chiral_conversion(bond_graph, disabled_bonds) is None:
```

Would prefer to avoid truthy checks in this code.

Reply:
This code was removed when we transitioned to a new approach in https://github.com/proteneer/timemachine/pull/1356/commits/546037d42cb12ff9af3303002f77d09c122c2ef9. I don't think there are any remaining cases of truthy checks.